app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>List of all Users</h1>
\ No newline at end of file
+<div class="users-list">
+    <% @users.each do |user| %>
+        <div class="user">
+            <div class="image">
+                <% if user.image.present? %>
+                    <%= image_tag(user.image, class: "img") %>
+                <% else %>
+                    <%= image_tag("no_image_found.png", class: "img") %>
+                <% end %>
+            </div>
+            <div class="details">
+                <a href="/users/<%= user.id %>"><h2><%= user.name %></h2></a>
+                <h4>Number of posts: <%= user.posts_counter %></h4>
+            </div>
+        </div>
+    <% end %>
```

Comment:
[OPTIONAL] In scenarios like this it is a good practice to use rails partials. This helps to keep your code DRY

Reply:
@fobadara Thank you for the review and the suggestion. I will definitely try learning more about partials and use them in the future.